server.js

```diff
@@ -1,16 +1,23 @@
-var express = require('express');
+var express =require("express");
 
 var app = express();
 
 var PORT = process.env.PORT || 3000;
 
+var db=require("./models");
+// console.log(db);
 //setup app to parse post data
 app.use(express.urlencoded({ extended: true }));
 app.use(express.json());
 
-var routes = require("./controllers/gamecontroller.js");
-app.use(routes);
+// var routes = require("./controllers/gamecontroller.js");
+// app.use(routes);
 
-app.listen(PORT,function(){
-    console.log('server running on port ' + PORT)
-});
\ No newline at end of file
+
+db.sequelize.sync({ force: true }).then(function() {
```

Comment:
Force true will rebuild the database and remove all the existing data. Is this something we want for development and we'll turn off later?

Reply:
Yeah,just want to keep it till the data base is stabilized.It can be removed after that.